packages/ibm-products/src/components/Tearsheet/Tearsheet.tsx

```diff
@@ -59,7 +59,7 @@ export interface TearsheetProps extends PropsWithChildren {
    *
    * See https://react.carbondesignsystem.com/?path=/docs/components-button--default#component-api
    */
-  actions?: ButtonProps<'button'>[];
+  actions?: ButtonProps<React.ElementType>[];
```

Comment:
We need to extend this, so that `label` can be provided as well. Otherwise we see type errors because we expect data to be passed in to construct the actions.